pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -145,12 +143,24 @@ let
       KEY_DH_OPERATIONS = whenAtLeast "4.7" yes;
 
       # needed for nftables
-      NF_TABLES_INET              = whenAtLeast "4.17" yes;
-      NF_TABLES_NETDEV            = whenAtLeast "4.17" yes;
-      NF_TABLES_IPV4              = whenAtLeast "4.17" yes;
-      NF_TABLES_ARP               = whenAtLeast "4.17" yes;
-      NF_TABLES_IPV6              = whenAtLeast "4.17" yes;
-      NF_TABLES_BRIDGE            = whenBetween "4.17" "5.3" yes;
+      # Networking Options
+      NETFILTER                   = yes;
+      NETFILTER_ADVANCED          = yes;
+      # Core Netfilter Configuration
+      NF_CONNTRACK_ZONES          = yes;
+      NF_CONNTRACK_EVENTS         = yes;
+      NF_CONNTRACK_TIMEOUT        = yes;
+      NF_CONNTRACK_TIMESTAMP      = yes;
+      NETFILTER_NETLINK_GLUE_CT   = yes;
+      NF_TABLES_INET              = whenAtLeast "4.19" yes;
+      NF_TABLES_NETDEV            = whenAtLeast "4.19" yes;
+      # IP: Netfilter Configuration
+      NF_TABLES_IPV4              = yes;
+      NF_TABLES_ARP               = whenAtLeast "4.19" yes;
+      # IPv6: Netfilter Configuration
+      NF_TABLES_IPV6              = yes;
+      # Bridge Netfilter Configuration
+      NF_TABLES_BRIDGE            = whenBetween "4.19" "5.3" yes;
```

Comment:
Can you change this to `module`?

Reply:
This code not work - `if whenBetween "4.19" "5.3" then yes else whenAtLeast "5.3" module;` don't know how to fix.
On the 5.3 kernel, this parameter is turned on automatically.

Reply:
This works:

```  
   NF_TABLES_BRIDGE            = mkMerge [
        (whenBetween "4.19" "5.3" yes)
        (whenAtLeast "5.3" module)
   ];
```

Reply:
Or would it also with 

```
   NF_TABLES_BRIDGE            = whenBetween "4.19" "5.3" yes;
```

result in a module getting build?

Reply:
I'm checking now

Reply:
In kernel 5.3
`CONFIG_NF_TABLES_BRIDGE=m`

Reply:
In kernel 5.2
`CONFIG_NF_TABLES_BRIDGE=y`